src/Explorer/Controls/SmartUi/SmartUiComponent.tsx

```diff
@@ -331,7 +331,7 @@ export class SmartUiComponent extends React.Component<SmartUiComponentProps, Sma
           onChange={(_, item: IDropdownOption) => this.props.onInputChange(input, item.key.toString())}
           placeholder={this.props.getTranslation(placeholderTKey)}
           disabled={disabled}
-          dropdownWidth="auto"
+          dropdownWidth={0}
```

Comment:
@hardiknai-techm this intended? 
@srinathnarayanan Does this affect you?

Reply:
hey, yes. Pls set it back to auto. what is the need for the change?

Reply:
revert back dropdownWidth="auto"